htdocs/commande/class/commande.class.php

```diff
@@ -1706,7 +1706,9 @@ public function addline($desc, $pu_ht, $qty, $txtva, $txlocaltax1 = 0, $txlocalt
 
 				if ($result > 0) {
 					$this->db->commit();
-					$this->lines[] = $this->line;
+					if (!isset($this->context["createfromclone"])) {
```

Comment:
This looks strange.
The method addline is to insert a new line.
So we must have the insert into the database and also into the memory object ->lines[];